cluster/saltbase/salt/salt-helpers/services

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
So I find this easier to reason than Salt, but I think the issue that you were encountering is that the service.running clauses in the salt files did not have a dependency to run after the .service file is written to disk.  I think if we just add the file watch for the .service files uniformly, you won't need this.